### PR TITLE
add inferno-flamegraph and separate makes

### PR DESCRIPTION
**Pull request by zeeshanlakhani**

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
adding the docker.mk path for version changes, but it can produce more noice. We may just want to force it via changes to the containers and versions, but I leave it as a q.

File: .github/workflows/packer_ansible.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
     - uses: actions/checkout@v1
 
     - name: build
-      run: make -f docker.mk build-packer-ansible
+      run: make -f packer-ansbile/docker.mk build
```
Comment (awangc) [inline .github/workflows/packer_ansible.yml]: typo

File: .github/workflows/packer_ansible.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
         DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
       run: |
         echo $DOCKERHUB_PASSWORD | docker login -u $DOCKERHUB_USERNAME --password-stdin
-        make -f docker.mk push-packer-ansible
+        make -f packer-ansbile/docker.mk push
```
Comment (awangc) [inline .github/workflows/packer_ansible.yml]: typo

File: .github/workflows/packer_ansible.yml (hunk 1 above)
Comment (zeeshanlakhani) [reply .github/workflows/packer_ansible.yml]: yep.

File: .github/workflows/packer_ansible.yml (hunk 2 above)
Comment (zeeshanlakhani) [reply .github/workflows/packer_ansible.yml]: oh jeez. thanks.

**Pull request merged**